DQMOffline/MuonDPG/interface/BaseTnPEfficiencyTask.h

```diff
@@ -0,0 +1,108 @@
+#ifndef BaseTnPEfficiencyTask_H
+#define BaseTnPEfficiencyTask_H
+
+/*
+ * \file BaseTnPEfficiencyTask.h
+ *
+ * \author L. Lunerti - INFN Bologna
+ *
+*/
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
```

Comment:
Why is this deprecated header included here?
(I have the impression that you can largely clean up the list of included headers...)

Reply:
I'm going to clean up the whole list of included headers (for all classes).